JobConfig/primary/TargetStopParticle.fcl

```diff
@@ -5,7 +5,7 @@
 #
 #include "Production/JobConfig/primary/StopParticle.fcl"
 
-physics.PrimaryPath : [ TargetStopResampler, @sequence::Common.generateSequence, @sequence::Common.g4Sequence, @sequence::Primary.PrimarySequence ]
+physics.PrimaryPath : [ TargetStopResampler, @sequence::Common.generateSequence, GenFilter, @sequence::Common.g4Sequence, @sequence::Primary.PrimarySequence ]
```

Comment:
GenFilter should be added to generateSequence.   It should default to FixedFilter, configured to return 'true', and overwritten as needed by specific generators.  I suggest changing the instance name to 'genfilter', to be consistant with 'generate'